articles/hooks/guides/create-delete-hooks-using-dashboard.md

```diff
@@ -39,7 +44,7 @@ Auth0 automatically enables the first Hook you create for an extensibility point
 
 ![List of Hooks](/media/articles/hooks/hooks-list.png)
 
-## Delete an Existing Hook
+## Delete an existing Hook
```

Comment:
Make this a separate doc "Delete Hooks Using the Dashboard" and add a when and why paragraph as an intro.